specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/SourceControls.json

```diff
@@ -230,6 +237,27 @@
             "required": true,
             "type": "string"
           },
+          {
+            "name": "x-ms-github-auxiliary",
+            "in": "header",
+            "description": "Github personal access token used for SourceControl.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "ignoreWorkflowDeletionFailure",
+            "in": "query",
+            "description": "Ignore Workflow Deletion Failure.",
+            "required": false,
+            "type": "boolean"
+          },
+          {
+            "name": "deleteWorkflow",
+            "in": "query",
+            "description": "Delete workflow.",
+            "required": false,
+            "type": "boolean"
+          },
```

Comment:
Update the examples to reflect these

Reply:
Update examples to reflect ignoreWorkflowDeletionFailure and deleteWorkflow.

Reply:
updated